llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -1621,8 +1621,15 @@ function(add_unittest test_suite test_name)
   # The runtime benefits of LTO don't outweight the compile time costs for tests.
   if(LLVM_ENABLE_LTO)
     if((UNIX OR MINGW) AND LINKER_IS_LLD)
-      set_property(TARGET ${test_name} APPEND_STRING PROPERTY
-                    LINK_FLAGS " -Wl,--lto-O0")
+      if(LLVM_ENABLE_FATLTO)
+        # When using FatLTO, just use relocatable linking.
+        set_property(TARGET ${test_name} APPEND_STRING PROPERTY
+                      LINK_FLAGS " -Wl,--no-fat-lto-objects")
+        set_property(TARGET ${test_name} APPEND_STRING PROPERTY COMPILE_FLAGS " -fno-lto")
```

Comment:
What's the motivation for including `-fno-lto` in compile flags here?

Reply:
This makes sure we use the non-LTO part of the build. It's probably pedantic to add them both, but it makes sure we choose the object file part and not the bitcode section.

Reply:
I'd omit it since it shouldn't be necessary.